Resources/Prototypes/_DV/Entities/Objects/Devices/CircuitBoards/Machine/production.yml

```diff
@@ -21,3 +21,84 @@
       Capacitor: 2
       Glass: 4
       Cable: 5
+
+- type: entity
+  parent: BaseMachineCircuitboard
+  id: EngineeringTechFabCircuitboard
+  name: engineering techfab machine board
+  description: A machine printed circuit board for an engineering techfab.
+  components:
+    - type: Sprite
+      state: engineering
+    - type: MachineBoard
+      prototype: EngineeringTechFab
+      stackRequirements:
+        MatterBin: 2
+        Manipulator: 2
+      tagRequirements:
+        GlassBeaker:
+          amount: 2
+          defaultPrototype: Beaker
+    - type: StealTarget
+      stealGroup: EngineeringTechFabCircuitboard
```

Comment:
i know you stole it from the upstream meme pr unindent all these components by 2 + make them have a basetechfab board with the requirements to not copy paste it